sehersarin/MSCI342-Term-Project
# Validation+tests

Ticket: S3T2

Description: authentication at log-in. I passed in name and type to dashboard, and I did an example of conditional rendering to differentiate user dashboard and worker dashboard.

Note: testing of this component will be done in another PR along with General T3

How to test: 

checkout this branch, then test the credentials of a student : johndoe@gmail.com, a1234. Check if the Header is red.
Then check the credentials of a worker : 
joshuabrooks@gmail.com, j1234. Check if the Header is blue.
Then check a faulty credential. Check if the dashboard does not appear.
